Update video_player compatibility (#10701)
flutter/website: src/content/cookbook/plugins/play-video.md

You must test network-hosted videos on physical iOS devices.
:::

### MacOS

> If you are using network-based videos, you will need to [add the

Reviewer: **issue:** Write in active voice per [GSG](https://developers.google.com/style/voice). 

```suggestion
If you use network-based videos, [add the
```

> ### MacOS
> 
> If you use network-based videos, [add the
> `com.apple.security.network.client`
> entitlement](https://docs.flutter.dev/platform-integration/macos/building#entitlements-and-the-app-sandbox)

Reviewer 2: ```suggestion
### macOS

If you use network-based videos, 
[add the `com.apple.security.network.client` entitlement][mac-entitlement].

[mac-entitlement]: platform-integration/macos/building#entitlements-and-the-app-sandbox
```

Author: TIL that macOS starts with a lower "M" I thought in the other readme that this would be a typo

> ### Web
> 
> > The Web platform does **not** support `dart:io`, so avoid using the `VideoPlayerController.file`
> > constructor for the plugin. Using the constructor attempts to create a
> > `VideoPlayerController.file` that will throw an `UnimplementedError`.

Reviewer 2: Why are these lines prefixed with an angle bracket?

```suggestion
### web

Flutter web does **not** support `dart:io`,
so avoid using the `VideoPlayerController.file` constructor for the plugin.
Using this constructor attempts to create a`VideoPlayerController.file`
that throws an `UnimplementedError`.
```

Author: I just took the formatting from the other readme file. I don't know exactly which markdown flavor is used here

> \* Different web browsers may have different video-playback capabilities (supported formats,
> autoplay...). Check [package:video_player_web](https://pub.dev/packages/video_player_web) for more
> web-specific information.

Reviewer 2: Why did the following line start with a "/*"? Is this being added to code?

```suggestion
Different web browsers might have different video-playback capabilities,
such as supported formats or autoplay.
Check the [video_player_web](https://pub.dev/packages/video_player_web)
package for more web-specific information.

[video_player_web]: {{site-pub-pkg}}/video_player_web
```

Author: Same here I just took the formatting as it was

> The `VideoPlayerOptions.mixWithOthers` option can't be implemented in web, at least at the moment.
> If you use this option in web it will be silently ignored.

Reviewer 2: ```suggestion
The `VideoPlayerOptions.mixWithOthers` option can't be implemented in web,
at least at the moment. If you use this option in web it will be silently ignored.
```
